core/src/main/java/org/springframework/security/authorization/method/AuthorizationReactiveMethodInterceptorBeanFactoryPostProcessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2002-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.authorization.method;
+
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.beans.factory.support.BeanDefinitionRegistry;
+import org.springframework.beans.factory.support.BeanDefinitionRegistryPostProcessor;
+import org.springframework.beans.factory.support.RootBeanDefinition;
+
+/**
+ * @author Evgeniy Cheban
+ */
+final class AuthorizationReactiveMethodInterceptorBeanFactoryPostProcessor
+		implements BeanDefinitionRegistryPostProcessor {
```

Comment:
The `AuthorizationReactiveMethodInterceptorBeanFactoryPostProcessor` cannot be moved to spring-security-config module because it is used in interceptors.

Perhaps when https://github.com/spring-projects/spring-framework/issues/22462 is fixed, we won’t need it anymore.
I'd suggest leaving it package-private for now, what do you think?